cfy_manager/components/nginx/nginx.py

```diff
@@ -408,12 +408,39 @@ def _configure_wait_on_restart_wrapper_service(self):
         # updating the ssl state as it required to restart nginx
         service.enable('wait_on_restart')
 
+    def _set_selinux_policies(self):
+        # Setup Cloudify SELinux policies for nginx
+        logger.info('Deploying SELinux policy for nginx...')
+
+        if MANAGER_SERVICE in config.get(SERVICES_TO_INSTALL):
+            # Enable nginx proxying for ports  3030, 8088, 8100
+            with TemporaryDirectory() as tmpdirname:
+                deploy(join(CONFIG_PATH, 'cloudify_manager.te'),
+                       join(tmpdirname, 'cloudify_manager.te'))
+                script_path = join(SCRIPTS_PATH, 'sepolicy_deploy.sh')
+                common.sudo(
+                    [script_path, tmpdirname,
+                     'cloudify_manager', '3000 8088 8100'])
+
+        if MONITORING_SERVICE in config.get(SERVICES_TO_INSTALL):
```

Comment:
You should probably install that policy anyway, since someone could install the monitoring service later, or as part of a three node cluster presumably?

Reply:
In case of the 3-node cluster, the policy would be installed then as a part of `cfy_manger install -c /path/to/manager_config.yaml`. Whereas for 9-node cluster we do not need nginx to communicate with those 3 ports on db or broker nodes.

Reply:
Fair enough, marking this as resolved then.